Updated tabs topic (#413)
IgniteUI/indigo-design-docfx: en/components/tabs.md

<img class="responsive-img" src="../images/tabs_fixed.png" srcset="../images/tabs_fixed@2x.png 2x" />

> <img class="responsive-img" src="../images/tabs_fluid.png" srcset="../images/tabs_fluid@2x.png 2x" />

Reviewer: the fluid image looks broken

## Tabs Amount

> For the majority of scenarios, the Tabs need to contain between two and four tabs. You can exclude a tab by setting it to ~No Symbol. If your case requires more tabs, you may want to consider a fluid mode and represent only the tabs in view.

Reviewer: ```suggestion
For the majority of scenarios, the Tabs need to contain between two and five tabs. You can remove a tab item by setting it to ~No Symbol. If your case requires more tabs, you may want to consider a fluid mode and represent only the tabs in view.
```

<img class="responsive-img" src="../images/tabs_2.png" srcset="../images/tabs_2@2x.png 2x" />
<img class="responsive-img" src="../images/tabs_3.png" srcset="../images/tabs_3@2x.png 2x" />
<img class="responsive-img" src="../images/tabs_4.png" srcset="../images/tabs_4@2x.png 2x" />

> 

Reviewer: add an image with five tabs

> ## Short Bar Tabs Content

Reviewer: ```suggestion
## Short Bar Content
```

> The short bar comes with **label** tabs but if you want to describe the tabs with an icon instead you have to set the label of each tab to ~No Symbol.

Reviewer: ```suggestion
The short bar comes with text content by default and if you want to show an icon instead, you have to assign a Material Icon to the `🔣 Icon` override and set the `🏷️ Label` to ~No Symbol for every tab in the bar.
```

<img class="responsive-img" src="../images/tabs_text.png" srcset="../images/tabs_text@2x.png 2x" />
<img class="responsive-img" src="../images/tabs_icons.png" srcset="../images/tabs_icons@2x.png 2x" />

> ## Tabs State
> 
> The Tabs consists of tabs that support Active, Inactive and Disabled states. In Tabs there is always one Active tab and an arbitrary number of Inactive and Disabled ones.

Reviewer: ```suggestion
## Tab Item State

Tab items support Active, Inactive and Disabled states. In Tabs there is always one Active tab and an arbitrary number of Inactive and Disabled ones.
```
